.eslintrc.js

```diff
@@ -74,6 +81,7 @@ module.exports = {
     //"dot-notation": "off",
     "eqeqeq": ["error", "allow-null"],
     "guard-for-in": "error",
+    //"max-classes-per-file": "off",
```

Comment:
@GianlucaGuarini to have a clean seperation of concern and basic OOP I would turn this to `error`

Reply:
I thought so too, but then I mused how many classes of us are in use and came to the number zero 🤔

Reply:
hopefully this will never change anytime soon. But the rule could be turned on if it doesn't break anything

Reply:
I'll leave it off - Less rules = Faster compilation